OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -653,6 +653,13 @@ public enum MovementType
 		Turn = 4
 	}
 
+	[Flags]
+	public enum MoveResult
+	{
+		CanRetry = 0,
+		SuggetGiveUp = 1,
```

Comment:
```suggestion
		SuggestGiveUp = 1,
```

Reply:
Or maybe
```suggestion
		CannotReachTarget = 1,
```

Reply:
I change the name of the enum here to `MovementStuck`, then `StuckByImmovable`.